src/commands/dev.ts

```diff
@@ -48,7 +48,7 @@ const command = defineCommand({
     // Prepare
     overrideEnv('development')
     const cwd = resolve(ctx.args.cwd || ctx.args.rootDir)
-    showVersions(cwd)
+    await showVersions(cwd)
```

Comment:
This was intentionally to block rest of build logic for informational information